internal/profiles/validate.go

```diff
@@ -22,15 +22,28 @@ func Validate() (err error) {
 		return err
 	}
 
-	// Make sure selected active profile is in the configuration file
 	activeProfileName, err := GetOptionValue(options.RootActiveProfileOption)
 	if err != nil {
 		return fmt.Errorf("failed to validate Ping CLI configuration: %w", err)
 	}
 
-	if !slices.Contains(profileNames, activeProfileName) {
-		return fmt.Errorf("failed to validate Ping CLI configuration: active profile '%s' not found in configuration "+
-			"file %s", activeProfileName, GetKoanfConfig().GetKoanfConfigFile())
+	profileName, err := GetOptionValue(options.RootProfileOption)
+	if err != nil {
+		return fmt.Errorf("failed to validate Ping CLI configuration: %w", err)
+	}
+
+	if profileName != "" {
```

Comment:
Lets check both instead of either with an if block.
It probably makes sense to read and validate the `profileName` first, then read and validate the `activeProfileName`.
This is because `Validate()` is mainly supposed to confirm configuration file is in a valid state so we always want to confirm `activeProfileName`.